src/app/docs/card/card-docs.component.html

```diff
@@ -0,0 +1,22 @@
+<sky-docs-demo-page
+  moduleName="SkyCardModule"
+  moduleSourceCodePath="src/app/public/modules/card/"
+  pageTitle="Card"
+>
+  <sky-docs-demo-page-summary>
+    The card module creates a small container to highlight important information. 
```

Comment:
Do you think it's worth including the point about related info? Maybe pull that part of the second sentence in to make this something like `Cards create small containers that highlight important information, and you group cards together to display information about related items.`? Or is that point implicit and unnecessary?